### ISSUE TITLE
Parsed css for page_detail container display doesn't work

### ISSUE THREAD

**Issue by juergmessmer** [rapila/cms-base#241]
No containers are displayed, no errors
use_parsed_css_in_config: true

**Comment by juergmessmer** [comment]
this setting also affects correct loading of page js: clicking on edit template and page type dialog doesn't work (Safari). Also changes in templates don't take effect when clearing the caches (Safari and Firefox).

**Comment by sabberworm** [comment]
Does this affect 1.7 or 1.5 as well or only master

**Comment by juergmessmer** [comment]
I’ll have to test it

**Comment by juergmessmer** [comment]
I can't test it anymore since release-1.5 is not compatible with my php 7 environment.

**Comment by sabberworm** [comment]
Tested with cms-full. Both branches (master and release-1.7) work fine.

<img width="531" alt="screen shot 2018-06-18 at 21 18 49" src="https://user-images.githubusercontent.com/47652/41557274-64b3ecd8-733d-11e8-9e93-20fc4cebbb82.png">

**Comment by sabberworm** [comment]
But there are many CSS3 (and beyond) features that PHP-CSS-Parser does not support. Maybe your site uses one of [those](https://github.com/sabberworm/PHP-CSS-Parser/issues/)?

**Comment by juergmessmer** [comment]
I don't understand all the issues, for example https://github.com/sabberworm/PHP-CSS-Parser/issues/125
Regarding background-position, I only use `background-position: left center;` not the suggested `background-position: top left, right bottom;`

**Comment by sabberworm** [comment]
> I don't understand all the issues, for example https://github.com/sabberworm/PHP-CSS-Parser/issues/125
> Regarding background-position, I only use `background-position: left center;` not the suggested `background-position: top left, right bottom;`

Well, then it does not apply to you, does it?

**Comment by sabberworm** [comment]
Also I think you mean sabberworm/PHP-CSS-Parser#123, not 125

**Comment by juergmessmer** [comment]
No, I meant the one related to "... declare color", I didn't understand that issue. The one related to background-position I checked and couldn't find any rules using the statements suggested in the related issue.
